spec/models/cart_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe Cart, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
```

Comment:
The Gem is... But in this case, you created a model on your application that has no specs.